charts/ceph-csi-cephfs/values.yaml

```diff
@@ -319,3 +319,5 @@ configMapName: ceph-csi-config
 # configMapKey:
 # Use an externally provided configmap
 externallyManagedConfigmap: false
+# Name of the configmap used for ceph.conf
+cephConfigMapName: ceph-config
```

Comment:
change the configmap name to avoid upgrade problems or else the user needs to pass different names for both cephfs and rbd by default?

Reply:
I think the name is consistent with other deployment options, plus it stays backwards compatible for users upgrading. Only people with multiple deployments of cephfs and/or rbd in the same namespace will need to rename.

Personally I would prefer that all resources were named after the release name to avoid external conflicts but that is a much bigger (and disruptive) change.